### ISSUE TITLE
Unable to buy a boat with Wynntils

### ISSUE THREAD

**Issue by TimeBomb** [Wynntils/Wynntils-Legacy#585]
### Version

1.12.3

### Brief Description

Unable to buy a boat from the boat purchase UI when at sailing boat.

### Steps to Reproduce

1. Go to sailing boat.
2. Right click sign to see world map with options to sail
3. In the bar of icons at the bottom of the screen, click the "Buy a boat" icon.
4. Nothing happens

### Expected Behaviour

Would expect to be able to purchase boat (and see the price it costs).

### Actual Behaviour

Nothing happens after clicking "Buy a boat"

### Additional Comments

With Wynntils disabled, reverting to the original Wynncraft boat UI, you are able to use that UI to purchase a boat without issue.

**Comment by IngeniousCoder** [comment]
Could be a Wynncraft Issue idk, bcuz boats are glitched currently so even if the purchase succeeds you can't use it

**Comment by TimeBomb** [comment]
Yeah there is a separate Wynncraft bug right now where you can't summon boats in the water. Boats do work if you use them on land and row them into the water, though.
That Wynncraft bug appears separate from this Wynntils bugs.
